### PR TITLE
Setup digital wallet onboarding

### PR DESCRIPTION
**Pull request by GiuliaTeggi**
### Description

Adding digital wallet onboarding steps with gov id not verified and verified flows. 

Placed timeline behind a feature flag (`timeline`) to hide it and add new feature flag to show the digital wallet onboarding (`digitalWallet`).

⚠️ feature flags will need to be seeded into the db before the launch.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: apps/web/app/[locale]/[event]/MyLifeEvents.tsx [hunk 1]
```diff
@@ -9,53 +9,95 @@ import { orderBirthCertificateRules } from "./[...action]/OrderBirthCertificate/
...
+    default:
+      throw new Error(`Unsupported workflow: ${flow}`);
+  }
 };
 
 async function getEvents() {
   "use server";
 
   return Promise.resolve([
+    // All the commented out events are now reachable via the burger menu
```
Comment (edge33) [inline apps/web/app/[locale]/[event]/MyLifeEvents.tsx:57]: should we delete this?

File: apps/web/app/[locale]/[event]/[...action]/GetDigitalWallet/ChangeDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+  const red = ds.colours.ogcio.red;
+
+  const errors = await form.getErrorsQuery(
+    userId,
+    routes.digitalWallet.getDigitalWallet.changeDetails.slug,
+    flow,
+  );
+
+  async function submitAction(formData: FormData) {
```
Comment (edge33) [inline apps/web/app/[locale]/[event]/[...action]/GetDigitalWallet/ChangeDetails.tsx:25]: non blocking, nit: I believe we should standardise the way we scaffold server action for better readability, decide whether we want to adopt guidelines for separate folder/dedicated path

**Review by edge33 — APPROVED** [review]
LGTM

File: apps/web/app/[locale]/[event]/[...action]/GetDigitalWallet/ChangeDetails.tsx (hunk 2 above)
Comment (GiuliaTeggi) [reply apps/web/app/[locale]/[event]/[...action]/GetDigitalWallet/ChangeDetails.tsx:25]: Sure, at the moment this is the way all form submit actions are setup for life events workflows. I don't have anything against anyone refactoring this at any point, it probably should be refactored as you say :)

**Pull request merged**